packages/mesos-modules/build

```diff
@@ -1,15 +1,29 @@
 #!/bin/bash
+set -o nounset -o pipefail
```

Comment:
these don't need to be set / are automatically set for all pkgpanda builds (https://github.com/dcos/dcos/blob/master/pkgpanda/build/__init__.py#L1173)